core/src/main/scala/com/spotify/featran/transformers/HashNHotEncoder.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.featran.transformers
+
+import com.spotify.featran.FeatureBuilder
+import com.twitter.algebird.HLL
+
+import scala.collection.SortedSet
+
+object HashNHotEncoder {
+  /**
+    * Transform a collection of categorical features to binary columns, with at most N one-values.
+    * Similar to [[NHotEncoder]] but uses MurmursHash3 to hash features into buckets
+    * to reduce CPU and memory overhead.
+    *
+    * Missing values are transformed to [0.0, 0.0, ...].
+    *
+    * When using aggregated feature summary from a previous session, unseen labels are ignored.
```

Comment:
Remove this since it only applies to regular encoders. Add `@param hashBucketSize`.